README.md

```diff
@@ -1,13 +1,13 @@
 
-**Title**: List of Prank Call Scripts
+**Title** : List of Prank Call Scripts
```

Comment:
Consider removing the extra space before the colon to maintain consistency with English punctuation rules. [medium]